Smartpad-iOS/Smartpad-iOS/Controller/SettingsViewController.swift

```diff
@@ -29,23 +35,18 @@ class SettingsViewController: UIViewController, UITextFieldDelegate{
         updateConnUI()
         changeNameField.delegate = self
     }
-    
+
+    // TODO: Ali what does this do
```

Comment:
@DJMaest If you could add descriptions for these on this pull request that would be great